### PR TITLE
renamed OVS domain and refactored

### PR DESCRIPTION
**Pull request by holmHin**

### CODE REVIEW

File: domain/ovs_domain.yaml [hunk 1]
```diff
@@ -86,32 +86,32 @@ components:
...
+    #     - type: object
+    #       properties:
+    #         UNLocationCode:
+    #           $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/UNLocationCode'
+    #   required:
+    #     - callbackUrl
+    # transportCallSubscription:
+    #   allOf:
+    #     - $ref: '#/components/schemas/subscriptionID'
+    #     - $ref: '#/components/schemas/transportCallSubscriptionBody'
```
Comment (HenrikHL) [inline domain/ovs_domain.yaml:114]:
````suggestion

```

Comment (HenrikHL) [reply domain/ovs_domain.yaml:114]: shouldn't we remove all these comments ;-)

File: domain/ovs_domain.yaml [hunk 2]
```diff
@@ -15,34 +15,34 @@ components:
...
+    #     - callbackUrl
     
-    scheduleSubscription:
-      allOf:
-        - $ref: '#/components/schemas/subscriptionID'
-        - $ref: '#/components/schemas/scheduleSubscriptionBody'
+    # scheduleSubscription:
+    #   allOf:
+    #     - $ref: '#/components/schemas/subscriptionID'
+    #     - $ref: '#/components/schemas/scheduleSubscriptionBody'
```
Comment (HenrikHL) [inline domain/ovs_domain.yaml:45]:
```suggestion
```

File: domain/ovs_domain.yaml [hunk 3]
```diff
@@ -15,34 +15,34 @@ components:
...
+    #     - type: object
+    #       properties:
+    #         startDate:
+    #           $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/startDate'
+    #     - type: object
+    #       properties:
+    #         carrierServiceCode:
+    #           $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/carrierServiceCode'
+    #   required:
+    #     - callbackUrl
```
Comment (HenrikHL) [inline domain/ovs_domain.yaml:40]:
```suggestion
```

File: domain/ovs_domain.yaml (hunk 1 above)
Comment (HenrikHL) [inline domain/ovs_domain.yaml:114]:
```suggestion
```

**Comment by HenrikHL** [comment]
Lets remove the comments :-)

**Comment by holmHin** [comment]
@HenrikHL, yes you are right, I have overseen those comments on the HUB. removed them

**Review by HenrikHL — APPROVED** [review]

**Pull request merged**